jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -3,332 +3,133 @@ layout: classic-docs
+1. Copy your existing `circle.yml` file into a new directory called `.circle` at the root of your project repository.
 
-This is because 1.0 uses LXC as a base container with a BTRFS file system. Recent versions of Docker don’t run properly in this environment, which means users can’t access Docker’s full featureset on 1.0.
+2. Rename `.circle/circle.yml` to `.circle/config.yml`.
 
-In 2.0, if your job uses Docker, we’ll run your job on a dedicated VM so you can access all of Docker’s features.
+3. Add `version: 2` to the top of the `.circle/config.yml` file.
 
-### Flexible Job Configuration
-1.0 infers your project type and automatically runs the most suitable test and build commands. Running builds are divided into well-defined steps that are performed in order. For example, 1.0 always saves your dependency cache before running tests.
-
-While this configuration can be powerful, there are some drawbacks. Maybe you want to disable inference. Or maybe you need to save the dependency cache _after_ running tests since the tests themselves create _more_ dependencies.
-
-In 2.0, jobs are broken into granular steps. You can compose these steps within a job at your discretion. This gives you greater flexibility to run your build the way you want.
-
-To learn more, please see the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/).
-
-### Custom Build Image
-In 1.0, you’re restricted to the build image CircleCI provides. In Linux builds, there are 2 images you can use: Ubuntu 12.04 and 14.04. While these images come with many languages and tools pre-installed, it’s frustrating if you need a version of a service or dependency that isn’t included.
-
-Maybe you want to use a different version of MySQL than the one included in either default image. Installing another version adds time and complexity to your builds.
-
-In 2.0, we support almost all public Docker images. You can also create a custom image and run jobs on that. You can even compose multiple images together (like MySQL 5.7 + Redis 3.2) and run jobs on them as if they were a single image.
-
-To learn more, please see the [Project Walkthrough]( {{ site.baseurl }}/2.0/project-walkthrough/).
-
-----
-
-# Configuration Equivalents
-CircleCI 2.0 introduces a completely new syntax in `.circleci/config.yml`. This article will help you convert your 1.0 `circle.yml` to a 2.0 `config.yml`.
-
-
-## Machine
-
-### Timezone
-
-**1.0**
-
-```YAML
-machine:
-  timezone: Europe/Dublin
-```
-
-**2.0**
-
-You can set the timezone in Docker images with the `TZ` environment variable. The value should be something like: `TZ="/usr/share/zoneinfo/America/Los_Angeles"`. A full list of available timezone options is [available on Wikipedia](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
-
-In your `.circleci/config.yml` it would look like:
+4. Replace `machine:` with the following two lines, nesting all of the following sections under `build`:
```

Comment:
per conversation with Nathan, this might be possible for some users, but machine isn't required, so I will change this to make sure users know they must have the jobs: build: line for 2.0.